engine/tasks.py

```diff
@@ -8,7 +8,8 @@ def run(self, game):
 		raise NotImplementedError("Abstract call.")
 
 
-class ResolutionTask:
+# This has to be a new-style class in order to be able to use super()
```

Comment:
Just a little bit further down:
"Just a heads up... with Python 2.7, and I believe ever since super() was introduced in version 2.2, you can only call super() if one of the parents inherit from a class that eventually inherits object (new-style classes).

Personally, as for python 2.7 code, I'm going to continue using BaseClassName.__init__(self, args) until I actually get the advantage of using super()."